SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1284,6 +1289,19 @@ void USpatialReceiver::ProcessQueuedResolvedObjects()
 	ResolvedObjectQueue.Empty();
 }
 
+void USpatialReceiver::ProcessQueuedActorRPCsOnEntityCreation(AActor* Actor, SpatialGDK::RPCsOnEntityCreation* QueuedRPCs)
+{
+	ensure(QueuedRPCs != nullptr);
```

Comment:
I think we agreed this should be check?
```suggestion
	check(QueuedRPCs != nullptr);
```